.github/ISSUE_TEMPLATE/crx-tut-request.yaml

```diff
@@ -0,0 +1,61 @@
+name: Extension tutorial request
+description: Use this form to request a new Chrome Extension tutorial
+title: "[EXT-TUTORIAL] <title>"
+labels: [content, extensions]
+assignees: amysteam
+body:
+- type: markdown
+  attributes:
+    value: "## Thank you for sharing your feedback. Please fill out this form so that we can better understand what extension tutorial you are looking for:"
+- type: textarea
+  attributes:
+    label: What would this extension do?
+    description: A concise description of how the extension would work.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: What lessons would this tutorial teach?
+    description: Top three extension concepts it will teach.
```

Comment:
I made the last suggestion before I read this. I'm not sure a tutorial should cover multiple concepts unless they're related some how.

Maybe something like: "Up to three related concepts it will teach."

Reply:
Let me think of an example:

An extension that:
- The main component is a content script
- Uses the Storage API to save state
- Sends messages to the service worker.